nightfall-deployer/contracts/ChallengesUtil.sol

```diff
@@ -76,9 +76,6 @@ library ChallengesUtil {
                 transaction.recipientAddress != ZERO ||
                 transaction.commitments[0] == ZERO ||
                 transaction.commitments[1] != ZERO ||
-                transaction.commitments.length != 1 ||
-                transaction.nullifiers.length != 0 ||
```

Comment:
Does removing this nullifiers check mean we now need to re include `nullifiers[0] === ZERO` and `nullifiers[1] === ZERO` in this conditional?

Reply:
Added nullifier zero check